build/utils/dependenciesutils.go

```diff
@@ -0,0 +1,68 @@
+package utils
+
```

Comment:
I suggest moving this file into the ```utils``` package undee the root of the project. There's probably no need for another ```utils``` package under ```build```.